systemtest/src/test/java/io/strimzi/systemtest/upgrade/OlmUpgradeST.java

```diff
@@ -47,64 +43,61 @@
     private final String producerName = "producer";
     private final String consumerName = "consumer";
     private final String topicUpgradeName = "topic-upgrade";
-    private final String olmUpgradeClusterName = "olm-upgrade-cluster-name";
-    private final int messageUpgradeCount =  50_000; // 10k ~= 23s, 50k ~= 115s
+    // clusterName has to be same as cluster name in examples
+    private final String clusterName = "my-cluster";
```

Comment:
Hmmm, this naming is not good for our parallelism stuff. I know that it has to match but can we somehow set some random cluster-name dynamically?

Reply:
You cannot run it in parallel. Same for `StrimziUpgradeST` and `StrimziDowngradeST`